### ISSUE TITLE
Add a size parameter

### ISSUE THREAD

**Issue by Turin86** [petems/petems-swap_file#82]
While defaulting to memorysize is nice, I miss an optional "size" parameter.

Is there a way around this limitation?

**Comment by petems** [comment]
This is possible to do for a while, but I should add an example to the docs

```puppet
swap_file::files { '5GB Swap':
  ensure       => present,
  swapfile     => '/mnt/swap.5gb',
  swapfilesize => '5GB',
}
```

https://github.com/petems/petems-swap_file/pull/84

**Comment by petems** [comment]
Closed in #84
